.github/pr-auto-comments.yml

```
@@ -0,0 +1,135 @@
+comment:
+  on-update: edit
+  header: |
+    Hi {{ prAuthor }}! Thank you for contributing to PyZn! ✨
+  footer: |
+    ---
+    > I'm a bot, and this is an automated comment 🤖
+  snippets:
+    - id: docs-changes
+      files:
+      - docs/**.md
```

yamllint

[warning] wrong indentation: expected 8 but found 6

(indentation)